.github/templates/linux_ci_workflow.yml.j2

```diff
@@ -306,6 +306,12 @@ jobs:
         run: |
           # Ensure the working directory gets chowned back to the current user
           docker run --rm -v "$(pwd)":/v -w /v "${ALPINE_IMAGE}" chown -R "$(id -u):$(id -g)" .
+      - name: Zip test reports for upload
+        if: always()
+        run: |
+          # Remove any previous test reports if they exist
+          rm -f test-reports-*.zip
+          zip -r "test-reports-${TEST_CONFIG}.zip" test -i '*.xml'
```

Comment:
qq: will `tar` run faster? (size is not the issue here, but # of small files right?)

Reply:
Potentially tar could run faster, just using zip here since zip is what were using in previous steps for artifacts